src/platforms/cocoa/index.mdx

```diff
@@ -15,7 +15,7 @@ The SDK can be installed using [CocoaPods](http://cocoapods.org), [Carthage](htt
 
 Support for
 
-- iOS >= 8.0
+- iOS >= 9.0
```

Comment:
would make sense to explain that Cocoa v5 is iOS 8 and Cocoa v6 is iOS 9? we have a single page for 2 SDK versions, this is confusing IMO, at some point either we'll be misleading users or even losing them due to broken docs

Reply:
The reason we dropped is that no one uses iOS 8 anymore anyway.
I'd say lets just document the current version (in this case 6.0)

Reply:
Granted we need to review API docs too once GA lands

Reply:
Don't we have the possibility to have docs for different SDK versions?

Reply:
That's ideal but generally more work. We just point users to upgrade at any issue which is less maintenance burden for us.

Lets try to stick to this for as long as we can.